### PR TITLE
Project: Restaurant Page

### PR DESCRIPTION
**Pull request by Samitti**
I have completed the Project Restaurant Page, kindly review my code.

### CODE REVIEW

File: src/contact.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+  return {
+    contactPageMethod,
+  };
+}());
+
+export {
+  // eslint-disable-next-line import/prefer-default-export
+  contactPage,
+};
```
Comment (Dandush03) [inline src/contact.js:36]: - [ ] Since you're exporting only one page, use export default

File: src/menu.js [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+import Menu3 from './3.png';
+import Menu4 from './4.png';
+import Menu5 from './5.png';
+import Menu6 from './6.png';
+import Menu7 from './7.png';
+import Menu8 from './8.png';
+
+
+// eslint-disable-next-line func-names
+const menuPage = (function () {
```
Comment (Dandush03) [inline src/menu.js]: ### -[ ] use arrow function ES6 Syntax

**Review by Dandush03 — CHANGES_REQUESTED** [review]
# Status: :red_circle: : Require Changes :red_circle: : 
### Great Work So Far :muscle: 
You must avoid at all costs disabling EsLint. (Most of the time is correct.)
Don't forget for this project ES6 Syntax is a must.
Check out my recommendation below, they apply for all places where the same error repeat, I will not point  out each time it happens 
# Happy Coding :nerd_face: :computer:

**Comment by Samitti** [comment]
I have made the changes rewquired by the previous review, but for some reason I couldn't understand the export default is not working for me so I have disabled the eslint error. kindly review my code .

**Review by rOluochKe — APPROVED** [review]
Hello @Samitti 
### Well done, you got it right :clap: The project has been approved. Congratulations :boom: :100:
### Merge and submit the project. Good luck with the next one.
- Happy coding!

**Pull request merged**